src/index.js

```diff
@@ -59,49 +97,41 @@ class WarmUP {
     /** Default options */
     this.warmup = {
       memorySize: 128,
-      name: 'warmup-plugin-' + this.serverless.service.service + '-' + this.options.stage,
+      name: this.serverless.service.service + '-' + this.options.stage + 'warmup-plugin',
```

Comment:
This line is the only actual change. Everything else comes from the other pull request. My bad, I should have done it in separate branches from the beginning...